op-node/node/node.go

```diff
@@ -568,6 +568,33 @@ func (n *OpNode) Start(ctx context.Context) error {
 		return err
 	}
 	log.Info("Rollup node started")
+
+	// If n.cfg.Driver.SequencerUseFinalized is true, sequencer cannot retrieve non-finalized L1 blocks.
```

Comment:
Maybe you meant this?
```suggestion
	// If n.cfg.Driver.SequencerUseFinalized is true, sequencer cannot use non-finalized L1 blocks as L1 origin.
```